Makefile

```diff
@@ -13,7 +13,7 @@ DATA_PATH ?= $(shell pwd)/data
 MJKEY_PATH ?= ~/.mujoco/mjkey.txt
 
 test:  ## Run the CI test suite
-test: RUN_CMD = nice -n 11 pytest -v -m 'not huge and not flaky' --durations=0
+test: RUN_CMD = nice -n 11 pytest -v -n 5 -m 'not huge and not flaky' --durations=0
```

Comment:
Why `pytest -n 5`?

Reply:
I found that this hits max load average on my 10-core workstation.

Reply:
Okay, I guess that's as good a reason as any.